src/dev/activation.ts

```diff
@@ -0,0 +1,230 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { ExtContext } from '../shared/extensions'
+import { createCommonButtons } from '../shared/ui/buttons'
+import { createQuickPick } from '../shared/ui/pickerPrompter'
+import { DevSettings } from '../shared/settings'
+import { FileProvider, VirualFileSystem } from '../shared/virtualFilesystem'
+import { Commands } from '../shared/vscode/commands2'
+import { createInputBox } from '../shared/ui/inputPrompter'
+import { Wizard } from '../shared/wizards/wizard'
+
+interface MenuOption {
+    readonly label: string
+    readonly description?: string
+    readonly detail?: string
+    readonly executor: (ctx: ExtContext) => Promise<unknown> | unknown
+}
+
+/**
+ * Currently contains all known developer tools.
+ *
+ * Options are displayed as quick-pick items. The {@link MenuOption.executor} callback is ran
+ * if the user selects an option. There is no support for name-spacing. Just add the relevant
+ * feature/module as a description so it can be moved around easier.
+ */
+const menuOptions: Record<string, MenuOption> = {
+    editStorage: {
+        label: 'Edit Storage',
+        description: 'VS Code',
+        detail: 'Edit a key in global/secret storage as a JSON document',
+        executor: openStorageFromInput,
+    },
+}
```

Comment:
New commands/options should be added here instead of in `package.json`.
Won't bother with name-spacing until the need is obvious.